source/process/release-process.md

```diff
@@ -187,6 +187,12 @@ If a bug fix release is required, run through the following steps again:
         - [Upgrade GitLab Mattermost to RC1](https://mattermost.atlassian.net/browse/PLT-3116)
         - [Push final build to GitLab Mattermost](https://mattermost.atlassian.net/browse/PLT-3117)
         - [Cut build and set up RC1 servers](https://mattermost.atlassian.net/browse/PLT-3115)
+    - Create [PR tracking spreadsheet](https://docs.google.com/spreadsheets/d/1YkRqyQk0Y4ZouV-SsXbSjbzGOXu2ZSPSd4XC_4LAErI/edit#gid=0) for next release.
+        - Populate with any PR's that have already went in for the next release.
```

Comment:
Perhaps reword:
`Populate with any PR's that have already been submitted for the next release.`